docs/custom.md

```diff
@@ -231,30 +233,29 @@ This workflow takes as input the entire "workflow parameters" entry and a list o
 We now need to define each task, which contains its own inputs and outputs, e.g., the task that creates the force field file:
 
 ```yaml
-'workflow2':
+"workflow2":
 ... ### I/Os
-  'tasks':
+"tasks":
 ... ### Other tasks
-  - 'm_def': 'nomad.datamodel.metainfo.workflow.TaskReference'
-    'name': 'create force field'
-    'task': '<path_to_mainfile>/create_force_field.archive.yaml#/data'
-    'inputs':
-    - 'name': 'workflow parameters'
-      'section': '<path_to_mainfile>/workflow_parameters.archive.yaml#/data'
-    - 'name': 'workflow script 2'
-      'section': '<path_to_mainfile>/workflow_scripts.archive.yaml#/data/Files/1/file'
-    'outputs':
-    - 'name': 'force field file'
-      'section': '<path_to_mainfile>/create_force_field.archive.yaml#/data/Files/0/file'
+- "m_def": "nomad.datamodel.metainfo.workflow.TaskReference"
+  "name": "create force field"
+  "task": "<path_to_mainfile>/create_force_field.archive.yaml#/data"
+  "inputs":
+    - "name": "workflow parameters"
+      "section": "<path_to_mainfile>/workflow_parameters.archive.yaml#/data"
+    - "name": "workflow script 2"
+      "section": "<path_to_mainfile>/workflow_scripts.archive.yaml#/data/Files/1/file"
+  "outputs":
+    - "name": "force field file"
+      "section": "<path_to_mainfile>/create_force_field.archive.yaml#/data/Files/0/file"
 ```
 
 This task is linked to the entry defined in `create_force_field.archive.yaml`. It takes as input: 1. the entire workflow parameters entry, defined in `workflow_parameters.archive.yaml`, and 2. The second file stored in the files list within the workflow scripts entry, defined by `workflow_scripts.archive.yaml`. The output of this task is the force field file, which is the first file stored in the file list of the create for field entry.
 
-
 You can now add the "create box" and "insert water" tasks to create the final workflow file:
 
 ??? success "`setup_workflow.archive.yaml`"
-    ```yaml
+`yaml
```

Comment:
This change no longer renders inside the admonition, nor does it produce a code block. Pls revert.

Reply:
Also, the quotation marks in this block don't match your other ones.